config/settings.d/bmc.yml.example

```diff
@@ -6,3 +6,9 @@
 # - freeipmi / ipmitool - requires the appropriate package installed, and the rubyipmi gem
 # - shell - for local reboot control (requires sudo access to /sbin/shutdown for the proxy user)
 #:bmc_default_provider: freeipmi
+#
+# Setting the log level below will cause rubyipmi to output logs to /tmp/rubyipmi.log
+# If you want the Rubyipmi logs to go into the smart_proxy logs you will need to set the main log level in settings.yaml to DEBUG
+# and disable the provider_log_level below
+# valid options are WARN, DEBUG, ERROR, FATAL, INFO, UNKNOWN
+#:provider_log_level: DEBUG
```

Comment:
Can you reword the whole description, it's a bit confusing. Use words comment/uncomment to describe desired behavior. Describe current state (when it's commented) and what happens when you uncomment.

Is really `/tmp` the best sane default value? I'd vote for something like `/var/log/smart-proxy/rubyimpi.log` instead.

I think the default configuration should be to use Smart Proxy logger with INFO/WARN level to see warnings and errors there. We can extend our `foreman-debug` script to collect this information then: https://github.com/theforeman/smart-proxy/blob/develop/extra/foreman-debug-proxy